core/domain/email_manager.py

```diff
@@ -242,14 +245,17 @@ def _send_email(
             the email.
         reply_to_id: str or None. The unique reply-to id used in reply-to email
             address sent to recipient.
+        recipient_email: str or None. Override for the recipient email.
```

Comment:
Explain in the docstring when this parameter is allowed to be used, to prevent other devs using it when they should not.

Reply:
Done.